tests/integration_tests/functional/test_topology.py

```diff
@@ -198,6 +200,12 @@ def test_cpu_topology(uvm_plain_any, num_vcpus, htt):
     """
     if htt and PLATFORM == "aarch64":
         pytest.skip("SMT is configurable only on x86.")
+
+    # TODO:Remove (or adapt) this once we unify the way we expose the CPU cache hierarchy on
+    # Aarch64 systems.
+    if version.parse(get_kernel_version()) >= version.parse("6.14"):
```

Comment:
nit: why don't we use `host_linux_version` of `global_props` that we have already imported?

Reply:
no particular reason. Just saw we were doing it like that in various other places.